weave-js/vite.config.ts

```diff
@@ -76,7 +76,23 @@ export default defineConfig(({mode, command}) => {
     },
     {find: /^react-vis$/, replacement: 'react-vis/dist/index.js'},
     {find: 'dagre', replacement: 'dagre/dist/dagre.min.js'},
-
+    {
```

Comment:
Installing segment packages installs a package called `type` which clashes in the namespace with an identically named package. So here we tell vite to manually resolve it, similar to the app.